Facebook.Unity/Assets/Facebook/Scripts/Mobile/UWP/WindowsFacebookLoader.cs

```diff
@@ -0,0 +1,23 @@
+﻿
+namespace Facebook.Unity.Mobile.UWP
+{
+    using Facebook.Unity;
+    using Facebook.Unity.Mobile;
+
+    internal class WindowsFacebookLoader : FB.CompiledFacebookLoader
```

Comment:
This should be instantiated in the FB.init method at 
https://github.com/facebook/facebook-sdk-for-unity/blob/master/Facebook.Unity/Assets/Facebook/Scripts/FB.cs#L214